### DISCUSSION TITLE
how to get performance for known object setting?

### DISCUSSION THREAD

**Discussion by roy881020** [fredzzhang/spatio-attentive-graphs#69] [Q&A] [answered]
Hello,

I want to know how I get a performances for known object setting in HICO-DET testing.

I just got a performance for default setting after doing test by your instruction.

Please let me know some of references or instructions that  I can check.

Thanks.
roy

**Comment by fredzzhang** [comment, marked as answer]
Hi @roy881020,

The python code we implemented does not compute mAP under the know object setting. Alternatively, you can cache the results in Matlab format following the instructions, and then evaluate using the HO-RCNN repo.

Fred.
